### PR TITLE
OP-1368 | Make Inventory status language independent

### PR DESCRIPTION
**Pull request by ArnaudFofou**
See OP-1368.
Paired with https://github.com/informatici/openhospital-core/pull/1460

### CODE REVIEW

File: src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java [hunk 1]
```diff
@@ -602,8 +602,9 @@ private JComboBox<String> getComboBox() {
 		if (statusComboBox == null) {
 			statusComboBox = new JComboBox<String>();
 			statusComboBox.addItem("");
-			for (InventoryStatus currentStatus : InventoryStatus.values()) {
-				statusComboBox.addItem(MessageBundle.getMessage("angal.inventory." + currentStatus));
+			List<String> statusList = medicalInventoryManager.getStatusList();
+			for (String status : statusList) {
+				statusComboBox.addItem(MessageBundle.getMessage(status));
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java]: Doesn't `getStatusList()` return a list of localized statuses?  Why another `getMessager()`?

File: src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java [hunk 2]
```diff
@@ -1783,12 +1783,14 @@ private JLabel getReferenceLabel() {
 	private JLabel getStatusLabel() {
 		if (statusLabel == null) {
 			if (inventory == null) {
-				String currentStatus = InventoryStatus.draft.toString().toUpperCase();
-				statusLabel = new JLabel(currentStatus);
+				String currentStatus = InventoryStatus.draft.toString();
+				String status = medicalInventoryManager.getStatusByKey(currentStatus);
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java:1785]: Doesn't `getStatusByKey()` return a localized value?   Why then the `getMessage()` in the next line?

File: bundle/language_en.properties
Comment (mwithi) [inline bundle/language_en.properties:1]: should all status label be near each other? this file should be in alphabetical order

**Review by mwithi — APPROVED** [review]
Apart the comment about `language_en.properties`, it LGTM.

**Review by dbmalkovsky — APPROVED** [review]
LGTM

**Pull request merged**